src/unfccc_ghg_data/unfccc_reader/Thailand/config_tha_bur3.py

```diff
@@ -446,22 +449,40 @@
                 "3.C.6",
                 "3.C.7",
             ],
-            "name": "Aggregate sources and non-CO2 emissions sources on land (Agriculture)",
+            # "name": "Aggregate sources and non-CO2 emissions sources on land (Agriculture)",
```

Comment:
Why are all the names commented out?

Reply:
Because they were not used and just there as comments. 
The old code used the names to fill the additional coordinate "orig_cat_name". But i almost never keep the original category name additional coordinate as it's basically always an n to 1 mapping to category codes and thus not a valid additional coordinate for primap2. The new code is more flexible regarding the additional coordinates to consider in aggregation and interprets all additional keys as names of additional coordinates. Where an additional coordinate is used I renamed the key to reflect the additional coordinate name ("orig_cat_name"), but where it's effectively just a comment, I just commented it without renaming.